source/common/http/utility.cc

```diff
@@ -435,11 +435,16 @@ Utility::parseHttp1Settings(const envoy::config::core::v3::Http1ProtocolOptions&
 
 void Utility::sendLocalReply(const bool& is_reset, StreamDecoderFilterCallbacks& callbacks,
                              const LocalReplyData& local_reply_data) {
+  absl::string_view details = "";
```

Comment:
```suggestion
  absl::string_view details;
```

nit: I think default init would be fine here?